datahub-web-react/src/app/AppProviders.tsx

```diff
@@ -15,11 +16,13 @@ export default function AppProviders({ children }: Props) {
         <AppConfigProvider>
             <UserContextProvider>
                 <EntityRegistryProvider>
+                <BrowserTitleProvider>
```

Comment:
Code styling: This component should be indented to match the others please!

Reply:
I will update it cause due to conflict